astropy/cosmology/flrw/base.py

```diff
@@ -1678,9 +1645,9 @@ def __post_init__(self):
         object.__setattr__(self, "_Ode0", 0)
         super().__post_init__()
         # Do some twiddling after the fact to get flatness
-        object.__setattr__(self, "_Ok0", 0.0)
-        object.__setattr__(
-            self, "_Ode0", 1.0 - (self._Om0 + self._Ogamma0 + self._Onu0 + self._Ok0)
+        self.__dict__["Ok0"] = 0.0
+        object.__setattr__(  # managed by a Property
```

Comment:
Why not manage it with a cached property?

Reply:
This is a type. It should say "managed by a Parameter". So it can't be a cached property (yet ?).